### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by ZiadElKhattabi**
- url: https://ziadelkhattabi.wordpress.com/2017/12/11/first-blog-post/
  title: BETWEEN THE COLORED LINES
 img: computer-programming.jpeg
  author_name: Ziad El Khattabi

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@
   title: Interview with a Teacher from Doon International School
   img: yashkumarverma-blog02.jpg
   author_name: Yash Kumar Verma
+  
+- url: https://ziadelkhattabi.wordpress.com/2017/12/11/first-blog-post/
+  title: BETWEEN THE COLORED LINES
+  img: computer-programming.jpeg
```
Comment (nikit19) [inline _data/blogs.yml:43]: Please upload this image

**Review by nikit19 — CHANGES_REQUESTED** [review]
Looks good just make this change

File: _data/blogs.yml (hunk 1 above)
Comment (ZiadElKhattabi) [reply _data/blogs.yml:43]: ok

Comment (ZiadElKhattabi) [reply _data/blogs.yml:43]: When I try to upload  it, it said that uploads are disabled

Comment (nikit19) [reply _data/blogs.yml:43]: Are you uploading in your forked repository?

**Comment by harshit98** [comment]
Hi, @ZiadElKhattabi Make sure you've cloned the forked repository on your machine.

**Review by akshatnitd — COMMENTED** [review]
@ZiadElKhattabi I can see that you have already forked the repo. You need to add the image in `img/blog/` folder in your forked repo.

**Review by ParthS007 — COMMENTED** [review]
@ZiadElKhattabi  as there are some conflicts. Please read README.md on how to add a blog and ask on Gitter channel if you face any difficulties.

**Comment by ParthS007** [comment]
Closing as a duplicate of #759

**Pull request closed without merging**